test/bin/build_images.sh

```diff
@@ -56,6 +56,47 @@ configure_package_sources() {
     done
 }
 
+# Read a macro from a blueprint file and expand it into the list of the
+# container images. The format for the macro is "RENDER_CONTAINER_IMAGES=version".
+# The version is necessary to query the release info from the appropriate RPM package.
+render_container_images() {
+    local -r blueprint_file="${1}"
+    # Exit when no macro is present in the blueprint
+    if ! grep -q ^RENDER_CONTAINER_IMAGES= "${blueprint_file}" ; then
+        return
+    fi
+
+    # Extract the version string
+    local -r version="$(grep ^RENDER_CONTAINER_IMAGES= "${blueprint_file}" | cut -d'=' -f2 | xargs)"
+    if [ -z "${version}" ] ; then
+        echo "Error: missing version string for the RENDER_CONTAINER_IMAGES macro in ${blueprint_file}"
+        exit 1
+    fi
+
+    # Find the microshift-release-info RPM with the specified version
+    local -r release_info_rpm=$(find "${IMAGEDIR}/rpm-repos" -name "microshift-release-info-${version}*.rpm" | tail -1)
```

Comment:
That output from find should probably be sorted, too.

I wonder if it would be less error prone to use the repository name instead of the version string? How do I express that I want the RPM being built from the PR, for example?

Reply:
I fixed the `find` command output to be sorted. 
For using repository names, I think the versions are covering most of the cases we need because they are "always" explicitly mentioned in the blueprints. So, the macro would be something like:
```
RENDER_CONTAINER_IMAGES="${SOURCE_VERSION}"
```

Reply:
Sounds good, I didn't notice that the templating took 2 passes.